lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -1072,6 +1166,14 @@ defmodule LightningWeb.WorkflowLive.Edit do
            :error,
            "Cannot delete a node in snapshot mode, switch to latest"
          )}
+
+      :presence_low_priority ->
+        {:noreply,
+         socket
+         |> put_flash(
+           :error,
+           "Cannot delete a node in low priority mode"
```

Comment:
Not sure we should bring the term 'low priority' to the front end, perhaps "Cannot edit workflow while someone else is currently editing"?

@christad92 thoughts?

Reply:
Done